addons/hr_expense_stripe/wizard/cardholder_wizard.py

```diff
@@ -0,0 +1,120 @@
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        res = super().create(vals_list)
+
+        for wizard in res:
+            if wizard.stripe_cardholder_id:
+                payload = {'account': wizard.company_id.stripe_account_id}
+                response = make_request_stripe_proxy(f'/api/stripe_issuing/1/cardholders/{wizard.stripe_cardholder_id}', payload, method='GET')
+                wizard.firstname = response['individual']['first_name']
+                wizard.lastname = response['individual']['last_name']
+
+                wizard.email = response['email']
+                wizard.phone_number = response['phone_number']
+
+                year = response['individual']['dob']['year']
+                month = response['individual']['dob']['month']
+                day = response['individual']['dob']['day']
+                wizard.birthday = date(year=year, month=month, day=day)
+
+                wizard.billing_address_country = self.env['res.country'].search([('code', '=', response['billing']['address']['country'])], limit=1).id
+                wizard.billing_address_city = response['billing']['address']['city']
+                wizard.billing_address_line1 = response['billing']['address']['line1']
+                wizard.billing_address_postal_code = response['billing']['address']['postal_code']
+            else:
+                # Try prefill from Employee
+                private_first_name, *private_last_name = (wizard.employee_id.name or '').split(' ')
+                wizard.firstname = private_first_name
+                wizard.lastname = ' '.join(private_last_name)
+
+                wizard.email = wizard.employee_id.email
+                wizard.phone_number = wizard.employee_id.phone
+                wizard.birthday = wizard.employee_id.birthday
+
+                wizard.billing_address_country = wizard.employee_id.country_id
+                wizard.billing_address_city = wizard.employee_id.private_city
+                wizard.billing_address_line1 = wizard.employee_id.private_street
+                wizard.billing_address_postal_code = wizard.employee_id.private_zip
+
+        return res
+
+    def action_save_cardholder(self):
+        self.ensure_one()
+        payload = {}
+        try:
+            payload= {
+                'account': self.company_id.stripe_account_id,
+                'billing': {
+                    'address': {
+                        'country': self.billing_address_country.code,
+                        'city': self.billing_address_city,
+                        'line1': self.billing_address_line1,
+                        'postal_code': self.billing_address_postal_code,
+                    }
+                },
+                'name': f'{self.firstname} {self.lastname}',
+                'email': self.email,
+                'phone_number': phonenumbers.format_number(phonenumbers.parse(self.phone_number), phonenumbers.PhoneNumberFormat.INTERNATIONAL),
```

Comment:
would fail
use something like
```python
self.employee_id._phone_format('mobile_phone', country=self.employee_id.country_id, force_format='E164'),
```